Core/Pixel.swift

```diff
@@ -257,7 +256,7 @@ public class Pixel {
     
     public static func fire(pixel: PixelName,
                             forDeviceType deviceType: UIUserInterfaceIdiom = UIDevice.current.userInterfaceIdiom,
-                            withAdditionalParameters params: [String: String?] = [:],
+                            withAdditionalParameters params: [String: String] = [:],
```

Comment:
I changed this interface slightly so it matches APIRequest.request() and since we weren't ever using optionals here